<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPRdemoClass</class>
 <widget class="QMainWindow" name="LPRdemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间/Time:</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="updateTimeLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="capLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>601</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openPushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>打开/Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色:</string>
    </property>
   </widget>
   <widget class="QLabel" name="licenseLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>54</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>车牌号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorLineEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="licenseLineEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>110</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeCostLineEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>160</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timeCostLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>54</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>耗时:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabe">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>识别状态:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stateLineEdit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>210</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>历史记录</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outpuPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="screenShootLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="stateComboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="statelabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>54</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>运行状态:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause/Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lprdemo.qrc"/>
 </resources>
 <connections/>
</ui>
